packages/lab/src/divider/Divider.tsx

```diff
@@ -0,0 +1,54 @@
+import { ComponentPropsWithoutRef, forwardRef } from "react";
+import { clsx } from "clsx";
+import { makePrefixer } from "@salt-ds/core";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+
+import DividerCss from "./Divider.css";
+
+export interface DividerProps extends ComponentPropsWithoutRef<"hr"> {
+  /**
+   * The divider orientation.
+   */
+  orientation?: "horizontal" | "vertical";
+  /**
+   * The variant to use. Options are 'primary', 'secondary' and 'tertiary'.
+   * 'primary' is the default value.
+   */
+  variant?: "primary" | "secondary" | "tertiary";
+}
+
+const withBaseName = makePrefixer("saltDivider");
+
+export const Divider = forwardRef<HTMLHRElement, DividerProps>(function Divider(
+  props,
+  ref
+) {
+  const {
+    className,
+    orientation = "horizontal",
+    variant = "primary",
+    ...rest
+  } = props;
+
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-divider",
+    css: DividerCss,
+    window: targetWindow,
+  });
+
+  return (
+    <hr
+      className={clsx(
+        withBaseName(),
+        [withBaseName(orientation)],
+        [withBaseName(variant)],
```

Comment:
if we pass orientation=vertical we would need to set aria-orientation=“vertical”. We could make this a div and add role=“separator”  as feels a misnomer to use he for vertical separator ?

Reply:
done, modified to div